### DISCUSSION TITLE
Version 1.6.3.0 - Gold Pans ReStocked

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/TheGoldStandard#66] [Announcements]
[![The Gold Standard (GOLD) Version][MOD:shd:stat]][MOD:url] [![KSP version][KSP:shd:stat]][KSP:url]
[![License CC BY-NC-SA 4.0][LIC:shd]][LIC:url]  ![AVC .version files][AVCVAL:shd]  

---

# Version 1.6.3.0 - Gold Pans ReStocked

- 22 Jan 2022
- Release for Kerbal Space Program [KSP 1.12.3]

<h4 style="border:0.5px solid Tomato; background-color: #bada55; color: #FF0000; text-align:center">
<b>DO A CLEAN INSTALL:</br> DELETE EXISTING INSTALLATION THEN RE-INSTALL</b></h4>
<p style="border:0.5px solid Tomato; background-color: #bada55; color: #FF0000; text-align:center">Download from <a href="https://www.curseforge.com/kerbal/ksp-mods/TheGoldStandard/files">CurseForge</a></p>

### Updated♻

* renamed /Patches --> Compatibility

### CargoPart Review

* volume review (math shown below)
* game volume is in Liters (L)
* closes #48 - Review CargoPart
* small (1.25m) tanks had their cargo volumes reduced a small amount

  #### Math

  |Ore Tanks |   |   |   | πr²h | 1000L = m3
  -- | -- | -- | -- | -- | --
    | diameter (m) | radius (m) | height(m) | volume m^3 | *1000
  large | 2.5 | 1.25 | 1.9000 | 9.3266 | 9326.61
  small | 1.25 | 0.625 | 1.9000 | 2.33165 | 2331.66
  radial | 0.625 | 0.3125 | 1.0000 | 0.3068 | 306.8
  
### missing drag cubes

* closes #54 -  LargeGoldTank and LargeGoldOreTank missing drag cube

### Linting  and file maintenance :deciduous_tree: 

* all part.cfg's
* add Dewar and zer0Kerbal to authors
* minor housekeeping  
* patch dusting (mostly removing construction dust (comments)) 

### Squad-Tanks patches

* rename [Squad.cfg] to [Squad-Tanks.cfg] 1.2.0.0
* add [ModuleCargoPart] to all
  * Radial: 340 (already there)
  * Small: 2350
  * Large: 9500
* update patch license
* closes #55 - Squad-Tank patches

### Squad patches

* rename [Patches.cfg] to [Squad.cfg] 1.2.0.0
* remove sneaky little `MODULE[ModuleResourceHarvester]`
* Undo localization on some items
  * ResourceName
  * resourceName
* update patch license
* closes #56 - Squad patches

### Update [StockalikeMiningExpansion.cfg] v1.1.1.0

* update patch license

### Update [SCANSat.cfg] 1.1.0.2

* rename [Scansat.cfg] to [SCANSat.cfg] 1.1.0.2
* update patch license
* linting

### Add Restock.restockwhitelist

* closes #6 - ReStock compliance
* closes #58 - Add Restock.restockwhitelist

### RadialTank packedVolume 

* currently is packedVolume = 325
* stock is packedVolume = 340
* change to equal stock
* closes #57 - RadialTank packedVolume

### Status

* Localizations
  * closes #44 - Localization - Spanish (Español) <es-es.cfg> - thank you Fitiales
  * closes #65 - Localization - Russian (Русский) <ru.cfg> - thank you to DirtyThongSniffer
* Issues 
  * updates #47 - us-en.cfg should be complete
  * closes #46 - Localization - English (United States)] <en-us.cfg>
  * closes - #60 - Version 1.6.3.0-release
  * closes - #61 - 1.6.3.0 Legal MumboJumbo
  * closes - #62 - 1.6.3.0 Update documentation
  * closes - #63 - Release 1.6.3.0-release
  * closes - #64 - 1.6.3.0 Social Media
* Bug
  * closes #5 - Tank costs 
  * closes #12 - Text issue with SMX - anything remaining isn't a Localization issue
  * closes #15 - Drill patches need updating?
  * closes #16 - Tanks ModuleCargoPart: packedVolume
  * updates #11 - us-en.cfg should be complete
  
### See More

- Changelog Summary for more details of changes : See [ChangeLog][MOD:chlog]  
- Discussions and news on this mod: See [Discussions][MOD:discu] or [KSP Forums][MOD:url]  
- Known Issues for more details of feature requests and known issues : See [Known Issues][MOD:known]  

### Localization

> [x] ![][EN] English  
> [x] ![][RU] Russian (Русский)  
> [x] ![][ES] Spanish (Español)  
> [ ] ***your translation here***
>
> HELP WANTED - See the [README in the Localization folder][MOD:local] for instructions for adding or improving translations. [GitHub][GitHub:url] push is the best way to contribute. *Additions and corrections welcome!* 

---

### How to support this and other great mods by [`zer0Kerbal`][LINK:zer0Kerbal]  

[![Support][PAYPAL:img]][PAYPAL:url] [![Patreon][PATREON:img]][PATREON:url] [![Github Sponsor][GSPONS:img]][GSPONS:url] [![Buy zer0Kerbal a snack][BMCC:img]][BMCC:url]  

---

<div style="border:0.5px solid Tomato; background-color: #BADA55; color: #FF0000; text-align:center">
<p><b>*red box below is a link to forum post on how to get support*</p>  
<a href = "https://forum.kerbalspaceprogram.com/index.php?/topic/83212-*">
  <p><img src = "https://i.postimg.cc/vHP6zmrw/image.png" alt="How to get support"></p></a>
</br>
<p>Be Kind: Lithobrake, not jakebrake! Keep your Module Manager up to date</p></div>

---

[MOD:url]:   https://forum.kerbalspaceprogram.com/index.php?/topic/203990-* "TheGoldStandard on the forums"
[MOD:chlog]: https://raw.githubusercontent.com/zer0Kerbal/TheGoldStandard/master/changelog.md "Changelog"
[MOD:discu]: https://github.com/zer0Kerbal/TheGoldStandard/discussions "Discussions"
[MOD:issue]: https://github.com/zer0Kerbal/TheGoldStandard/issues "Issue Tracker"
[MOD:known]: https://github.com/zer0Kerbal/TheGoldStandard/wiki/Known-Issues "Known Issues"
[MOD:licns]: https://github.com/zer0Kerbal/TheGoldStandard/blob/master/LICENSE "Repo License"
[MOD:local]: https://github.com/zer0Kerbal/TheGoldStandard/blob/master/GameData/TheGoldStandard/Localization/readme.md "Localization"

[LINK:localization]: https://github.com/zer0Kerbal/TheGoldStandard/blob/master/GameData/TheGoldRush/Localization/readme.md "Localization"

[MOD:shd:stat]:  https://img.shields.io/badge/TheGoldRush!%20version-1.6.2.0-BADA55.svg?style=plastic&labelColor=darkgreen "1.6.3.0"

[CRSFG:url]: https://www.curseforge.com/kerbal/ksp-mods/TheGoldStandard/files "Curseforge"
[GITHUB:url]: https://github.com/zer0Kerbal/TheGoldStandard/ "GitHub"

[KSP:url]:      http://kerbalspaceprogram.com/ "Kerbal Space Program"
[KSP:shd:stat]: https://img.shields.io/badge/KSP-1.12.2-blue.svg?style=plastic&logo=data:image/webp;base64,UklGRpAGAABXRUJQVlA4TIQGAAAvH8AHEE0obNsGDakwXkT/Q4chj76jn1yYjSRjH6H+6xQqRUkkSc5ckHD+NUUVhx4+RNu2bTTl/3OTVhD6H8u6fF8dDYm40CK7N0CjAxK52rYtjfQRAvGNjbu7V+vuLp2fwhzA6gG4li6de+nuO+4uJBlkwoTA//90kENgtZZs27ZpO2Oufa5vbNsq2Sw6qaa9r3n/kZJTex/wSrFt27o62kuSJNuqrdTa1+/7uLu7uzQZAk0mRN/mAz2CAbi723vvy5WztxxIAAiw+dh2bdvWZJuT12y1bdu2bdu2bbs3AQABlsED+y50lDqLE4pf/Uxe8KO77HFes9hvxu1p3O2Q09LVg0NVjo7Z5U6AvwBSOfivKdQTujOnXNG8hzNhqYSMMHJ+MwWWyTSjNUoP1jWHp1ZNmgB8TMH3Tl33mvoVb8uIj3umyIJLsAmYoCIqUCVMCN5WrW78Qi+AkeaGOkWFk3QbRmIWHpIZSURXDxhJVlJiYZ6kLEgeEjIRCBmxebhCTWlTJNaXLystGysVToPOAFJJjqSPNOXftSVbXPQjxrzVaFNzXvBonWJlonIVTJabAfjZ0Jxufl5GPntEHKjB7PIXtg6eYHywJn3hTdijjXUain94KjfCfg1hM3AZRR4N16VMXBrsDVYDCjOIMmGFsAab8rdY7+kIw0BgtRgL1DKJeuNZdQ9f5RAGGfOQfZxexj7t37RkjNp4rPDIHOJVyPBF+XHKsnQk4SWRBl8Wot74WtkBqidQggbyj2vW1WEFG6JfUZ8UxhVzOmAUdKTYZ3yxR2QwIIAQdA9YgiRBZWyCguiOER9y9IauiyGqR0EayVLiEVwmXmF5+Z2jfohj8i1q8ybRahCBAXZBCBUqLgAACQFzp5M/o5wzCDcCJZEOF9EzggM7NEd18c1Q85gPBldtg6mwB1N+hzyTSFp5jWM+WpCRBEBAjIaUxJI80o3s6vDYJ7gNXuAUfOacvsLvLhNIMaMwh8GccsXjtuiK+wkxZ/kVdOCzoCeAOn+N7H1rU1YT35MgAWY9yD5wMWI7j7G976bSh8LxDx2jI3IJRAnJ+RQQIU7gABnLd3vKSyMAd71mq7HgV3AJ+Sxh5Bz5O0z/gbTiNT8DxQssrrv7Uyx9Y7q6BQVUoIGonF/FfoC/RqwVHlviCRSABDvLaCUfZlz1cNENn6adNTz+AF8v8KWQAFL+QMkpCthFkpvvq7+2/to6AIBhyvPdrUr3g1XwBEJPMeP/98SxWWvs3nHIhgNnsY1RWVyw+Guh5IZLgj9WzPi/iEk9gPPGLAni3aRlsqlGQ9jP4z9Q/s/xi3dw486abm6Bqi2l3qu9e9tQsdVMdG4zbVYqp5wJcBSA2W6sNmkwxuNk+nw3r4MFi/4EkTAhiRFk70jkqNDiWbDK65OLIn0s0wD/ucAp3XwVSsSNBAqjFUIXwFE0MihZQ6utQYV2oR+tShO7Ad6bswY3xd7qe0VrVxK9ZjueQy4TPnF8MCQGjoxSugSFgFMV4LiVwgsn/i+gXZ2FzplAduxnn0/OlW0uqf+M64MRPqzwrl+PnpKQSaXS49Ui7n2/ctFtHtAtiprzxq6WniwESvW5yUG1Xx6/8Hx8NKELwKZK15pV/EvXvm9ZMlx0aKUO98iUPaWvst/n8ZbOmkvGhcOQRWf5zj9dk9cfpad5oHN3Rns/wsuvy2puxz1Ziu96Q7/SOWoROzvNzFo5Z5+1BDej3OjQ/XymEkW9jr0em5g5SdX8VC2gf9xJb/RWCC5bIKWDgWcYf+K9Kje3zbQBh/F448wMLoICeUyJ330nXlPmawiRT/sblG4vWrbErgQaMzYbZcwbhSaNrwH+Tqa04jqrd3JZTvwbFxHFSVMAv5UZdEq+tQUupcis/5+MZNsxk9b8TPa7cMqdzzrh9FtD5v+vPACvJy7nDT69IP/Yx6EywGdTFsD5iU7bqkovJogzTjQm3iFTyp4jV4bjVKdcnv5/JrhokmpnGAIA4D/AXYCfVgoBXnrDkCqqCHRG529HeYB51Jy1z6nlW/gnVmzyxmVHxnQrxXxelcI0yN85udPl+//t2rzKzA+oluPTNjp6qY1PVduFVdo8ya+8E6p8KOZR+bLj6Vju9oi5dar0erTS8Z1x3/IITU3vyDRLiZWBZVH6CbqURTeLptD3pEPIR4W4QlHfTnRJzZBRJ8MlI8LmmEXLAdAxsqIYbSDGTt65GfF0cUL6aQQ= "Kerbal Space Program"  
  
[AVCVAL:shd]:    https://github.com/zer0Kerbal/Pteron/workflows/Validate%20AVC%20.version%20files/badge.svg?style=plastic&labelColor=black 

[LIC:url]:  https://creativecommons.org/licenses/by-nc-sa/4.0/ "CC-BY-NC-SA-4.0"
[LIC:logo]: https://licensebuttons.net/i/l/by-nc-sa/transparent/33/66/99/76x22.png "CC BY-NC-SA 4.0"
[LIC:shd]: https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/zer0Kerbal/TheGoldStandard/master/json/license.json "CC BY-NC-SA 4.0"

[PAYPAL:img]: https://img.shields.io/badge/Buy%20me%20some%20-LFO-00457C?style=for-the-badge&logo=paypal&labelColor=FFDD00 "PayPal"
[PAYPAL:url]: https://www.paypal.com/donate?hosted_button_id=DC22YHMEJREKL "PayPal"
[PATREON:img]: https://img.shields.io/badge/Patreon%20-be%20a%20Patron-FF424D?style=for-the-badge&logo=patreon "Patreon"
[PATREON:url]: https://www.patreon.com/bePatron?u=23390503 "Patreon"
[GSPONS:img]: https://img.shields.io/badge/Github%20-Sponsor-EA4AAA?style=for-the-badge&logo=githubsponsors "Github Sponsors"
[GSPONS:url]: https://github.com/sponsors/zer0Kerbal "Github Sponsors"
[BMCC:img]: https://img.shields.io/badge/Buy%20Me%20a%20-Snack!-FFDD00?style=for-the-badge&logo=buymeacoffee "Buy Me A Snack"
[BMCC:url]: https://buymeacoffee.com/zer0Kerbal "Buy Me A Snack"

[EN]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/American-flag-sm.png "American English"
[BR]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Brazilian-flag-sm.png "Brasil"
[CN]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Chinese-flag-sm.png "中文"
[DE]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/German-flag-sm.png "Deutsch"
[ES]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Spanish-flag-sm.png "Español"
[FR]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/French-flag-sm.png "Français"
[IT]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Italian-flag-sm.png "Italiano"
[JA]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Japanese-flag-sm.png "日本語"
[KO]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/South-Korean-flag-sm.png "한국어"
[ME]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Mexican-flag-sm.png "Mexicano"
[NL]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Dutch-flag-sm.png "Dutch"
[NO]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Norwegian-flag-sm.png "Norsk"
[PO]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Polish-flag-sm.png "Polski"
[RU]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Russian-flag-sm.png "Русский"
[SW]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Swedish-flag-sm.png "Svenska"
[TW]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Taiwanese-flag-sm.png "国语"
[HA]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/zed'K/Localization/img/Hawaii-flag-sm.png "ʻŌlelo Pākē"

[LINK:zer0Kerbal]:     https://forum.kerbalspaceprogram.com/index.php?/profile/190933-zer0kerbal/ "zer0Kerbal"

[HERO:0]: https://raw.githubusercontent.com/zer0Kerbal/TheGoldStandard/master/img/TGS-Black.png "The Gold Standard Flag"

## What's Changed
* Add Spanish (Español) <es-es.cfg> by @Fitiales in https://github.com/zer0Kerbal/TheGoldStandard/pull/51
* Add Russian (Русский) <ru.cfg> by @DirtyThongSniffer in https://github.com/zer0Kerbal/TheGoldStandard/pull/52
* 1.6.3.0 - Gold Pans ReStocked by @zer0Kerbalin https://github.com/zer0Kerbal/TheGoldStandard/pull/59

## New Contributors
* @Fitiales made their first contribution in https://github.com/zer0Kerbal/TheGoldStandard/pull/52
* @DirtyThongSniffer made their first contribution in https://github.com/zer0Kerbal/TheGoldStandard/pull/52
**Full Changelog**: https://github.com/zer0Kerbal/TheGoldStandard/compare/1.6.2.1...1.6.3.0-release

<hr /><em>This discussion was created from the release <a href='https://github.com/zer0Kerbal/TheGoldStandard/releases/tag/1.6.3.0-release'>Version 1.6.3.0 - Gold Pans ReStocked</a>.</em>

**Comment by MystLeissa** [comment]
What a name: "DirtyThongSniffer" - but then with the war I don't guess beggars can be choosers when it comes to Russian translators :P

**Reply by zer0Kerbal** [reply]
considering the sheer number of translations issues I have open - hundreds, won't be choosy about who does what as long as it is done right. 👍 

![image](https://user-images.githubusercontent.com/39887717/166166721-9f82468c-6a1c-4128-a362-d757c7f41394.png)

**Reply by zer0Kerbal** [reply]
@MystLeissa 
Just had one person provide Russian translations for _scores_ of my addons. 👍 🥇
